cluster-type/src/lib.rs

```diff
@@ -38,16 +48,32 @@ impl ClusterType {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum ParseClusterTypeError {
+    /// Unrecognized cluster type
+    Unrecognized,
+}
+
+impl core::error::Error for ParseClusterTypeError {}
```

Comment:
If we're using core::error::Error, this crate also needs to declare `rust-version = "1.81.0"` in Cargo.toml.

I'm not sure we totally need this error though